component/component/vshn_minio.jsonnet

```diff
@@ -25,6 +25,39 @@ local connectionSecretKeys = [
 
 local promRuleMinioSLA = prom.PromRuleSLA(params.services.vshn.minio.sla, 'VSHNMinio');
 
+
+local prometheusRule = prom.GeneratePrometheusNonSLORules(
+  'minio',
+  'minio',
+  [
+    {
+      name: 'minio-diskOffline',
+      rules: [
+        {
+          alert: 'minioDiskOfflineCritical',
+          annotations: {
+            description: "Minio server reports it's disk is down - {{ $labels.name }} in namespace {{ $labels.label_appcat_vshn_io_claim_namespace }}.",
+            summary: 'Minio disk is offline for at least 30 seconds.',
+          },
+
+          expr: 'minio_cluster_drive_offline_total != 0',
```

Comment:
Do we actually scrape this? Minio's metrics endpoint is behind authentication.

But it would be a very good alert nonetheless. So I would keep it either way. If/when we add the Minio metrics in the future will already have the alert.

Reply:
actually we're scraping those metrics thanks to @TheBigLee 
https://github.com/vshn/appcat/blob/master/pkg/comp-functions/functions/vshnminio/minio_deploy.go#L271

how it works without auth - no idea :D 

we're able to access those metrics (at least most of them) via:
https://console.lab-cloudscale-rma-0.appuio.cloud/monitoring/query-browser?query0=minio_cluster_drive_offline_total

Minio inside metrics gives us great potential for Grafana charts as it provides a lot of valuable insides, but very little for active alerting imho

Reply:
Even better! I thought we didn't have this yet because of the auth, but I like to be surprised positively :D